internal/cli/apps.go

```diff
@@ -209,9 +217,16 @@ auth0 apps delete <id>
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 0 {
-				if err := appID.Ask(cmd, &inputs.ID); err != nil {
+				err := appID.Pick(cmd, &inputs.ID, cli.appPickerOptions)
+				if err != nil {
 					return err
 				}
+
+				if inputs.ID == "" {
```

Comment:
Why the nil check if the options function already returns an error if there are no apps?